src/action/mod.rs

```diff
@@ -17,6 +17,8 @@
 
 pub mod metadata;
 pub mod startup;
+pub mod list_directory;
+// pub mod file_stat;
```

Comment:
Commented-out code should not be checked into the repository.

Reply:
Fixed.